scrapers/LewRubens.yml

```diff
@@ -0,0 +1,14 @@
+name: LewRubens
+
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - lewrubens.com/x-new/new-set.php?setid=
+    scraper: sceneScraper
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //meta[@name="twitter:title"]/@content
+      Details: //meta[@name="twitter:description"]/@content
+      Image: //meta[@name="twitter:image"]@content
```

Comment:
typo for the image part and the last updated comment is missing
```suggestion
      Image: //meta[@name="twitter:image"]/@content
# Last Updated February 6, 2021
```

Reply:
Fixed in a new commit.